target/linux/mvebu/base-files/etc/board.d/02_network

```diff
@@ -50,16 +50,16 @@ marvell,axp-gp)
 	ucidef_set_interface_lan "eth0 eth1 eth2 eth3"
 	;;
 solidrun,clearfog*a1)
-	# eth0 is standalone ethernet
-	# eth1 is switch (-pro) or standalone ethernet (-base)
-	# eth2 is SFP
-	ucidef_set_interfaces_lan_wan "eth1" "eth0 eth2"
+	# eth2 is standalone ethernet
+	# eth0 is switch (-pro) or standalone ethernet (-base)
+	# eth1 is SFP
+	ucidef_set_interfaces_lan_wan "eth0" "eth2 eth1"
```

Comment:
Does having two interfaces assigned to wan even work? WAN is no bridge, so I would expect this to be broken.
For reference: Change was introduced in https://github.com/openwrt/openwrt/commit/d41a441bb613537916460542eb89d517d1856b5d

Can you provide any evidence on the eth0/eth1 swap?

Reply:
1. multiple wan interfaces is supported, a bridge will be created. Here an extract from /etc/config/network on openwrt-19.07-rc2 which has not been configured yet:
´´´
root@OpenWrt:/# cat /etc/config/network 
...
config interface 'wan'
        option type 'bridge'
        option ifname 'eth0 eth2'
        option proto 'dhcp'
´´´

2. there is evidence for swapping in dmesg (it is how I knew to search for it):
´´´
[    5.614967] mvneta f1030000.ethernet eth0: renamed from eth1
[    5.664899] mvneta f1034000.ethernet eth1: renamed from eth2
[    5.704852] mvneta f1070000.ethernet eth2: renamed from tmp0
´´´

Reply:
Ah, config_generate is smarter than I thought.